yt/yt/server/queue_agent/queue_controller.cpp

```diff
@@ -1144,23 +1196,60 @@ class TOrderedDynamicTableController
             }
         }
 
-        //! Updates partition contexts in accordance with the offsets of vital consumers.
+        //! Updates partition contexts in accordance with the offsets of vital consumers and exports.
         //! Only affects the minimum trimmed row count.
-        void HandleVitalConsumers()
+        void HandleVitalConsumersAndExports()
         {
+            auto handleQueueExportByPartitionAndReturnMinTrimmedRowCount = [this] (TPartitionTrimContext& partitionContext) -> std::optional<i64> {
+                if (QueueExportProgress.empty()) {
```

Comment:
The `for` cycle over an empty map + default value of `minTrimmedRowCount` will produce the same result.